packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -474,6 +474,28 @@ describe('<Autocomplete />', () => {
       expect(textbox).toHaveFocus();
     });
 
+    it('should keep listbox open on pressing left or right keys when inputValue is not empty', () => {
```

Comment:
Can we also add test that will test that the listbox is closed if the input value is empty?

Reply:
Yes sure!